<?xml version="1.0"?>
<launch>
    <arg name="param_name" default="event_command_system.yaml"/>
    <node pkg="event_command_system" type="event_command_system.py" name="event_command_system_node" respawn="true" respawn_delay="5" output="screen">
        <rosparam file="$(find event_command_system)/params/$(arg param_name)" command="load" />
    </node>
</launch>